src/test/java/com/cag/cagbackendapi/integration/UserControllerIntegrationTests.kt

```diff
@@ -129,13 +129,13 @@ class UserControllerIntegrationTests {
 
     @Test
     fun registerUser_nullEmailAndFirstNameAndLastNameAndPassword_400BadRequest() {
-        val nullEmailUser = UserDto(null, null, null, null, true, null, null, true)
+        val nullEmailUser = UserRegistrationDto(null, null, null, null, true)
 
         val headers = HttpHeaders()
         headers.set("authKey", validAuthKey)
         val request = HttpEntity(nullEmailUser, headers)
 
-        val errorDetailsResponse = testRestTemplate.postForEntity("/user/register", request, ErrorDetails::class.java)
+        val errorDetailsResponse = testRestTemplate.exchange("/user/register", HttpMethod.POST, request, ErrorDetails::class.java)
```

Comment:
I have so many fixes to make for our testing. We really need that utility service that has methods to add users and profiles and stuff so that all of our update and delete and get usr tests don't have to write the same code to register a user and profile before testing